src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -218,10 +218,11 @@ private static ExtRequestTargeting targeting(ExtBidRequest requestExt) {
     /**
      * Extracts usepbsrates flag from {@link ExtBidRequest}.
      */
-    private static boolean usepbsrates(ExtBidRequest requestExt) {
+    private static Boolean usepbsrates(ExtBidRequest requestExt) {
         final ExtRequestPrebid prebid = requestExt != null ? requestExt.getPrebid() : null;
         final ExtRequestCurrency currency = prebid != null ? prebid.getCurrency() : null;
-        return currency != null ? currency.getUsepbsrates() : false;
+        final Boolean usepbsrates = currency != null ? currency.getUsepbsrates() : null;
+        return usepbsrates != null;
```

Comment:
you need to return usepbsrates. bc this method (according to documentation above) shoud only **extract**